scripts/generate-sdk/generate-sdk.sh

```diff
@@ -1,14 +1,13 @@
 #!/bin/bash
-# This script generates the SDK API modules
+# This script clones the SDK repo and updates it with the generated API modules
 # Pre-requisites: Java, goimports, Go
 set -eo pipefail
 
 ROOT_DIR=$(git rev-parse --show-toplevel)
 GENERATOR_PATH="${ROOT_DIR}/scripts/bin"
 GENERATOR_LOG_LEVEL="error" # Must be a Java log level (error, warn, info...)
-PREPARE_SDK_PATH="${ROOT_DIR}/prepare-sdk"
-SDK_PATH="${ROOT_DIR}/sdk"
-SDK_REPO="https://github.com/stackitcloud/stackit-sdk-go.git"
+SDK_REPO_LOCAL_PATH="${ROOT_DIR}/sdk-repo-cloned"
```

Comment:
I think the folder name is actually a bit misleading, because it is not just the repo cloned, it will also contain the updated generated SDK